src/Plugin.php

```diff
@@ -92,6 +92,9 @@ public static function init() {
       add_filter('acf/fields/relationship/query/key=field_spare_parts', __CLASS__ . '::acf_relationship_filter');
       add_filter('acf/fields/relationship/query/key=field_other_accessories', __CLASS__ . '::acf_relationship_filter');
     }
+    add_action('wp_ajax_related_accessories_backlinks', __NAMESPACE__ . '\Admin::wp_ajax_related_accessories_backlinks');
+    add_action('load-post.php', __NAMESPACE__ . '\Admin::related_accessories_backlinks_meta_box_setup');
```

Comment:
This is an odd hook name, given that we're operating on the `edit.php` page. Are you sure that this is the common way to implement something like this?  How did you find the `load-post.php` hook?

Reply:
https://developer.wordpress.org/reference/hooks/load-pagenow/ 
https://www.smashingmagazine.com/2011/10/create-custom-post-meta-boxes-wordpress/

Is there a better way? If so - which hook would you suggest and why?